src/core/libmaven/mzUtils.cpp

```diff
@@ -291,13 +227,16 @@ data		1-D array[ns] of smoothed data
     }
 
     float median(float* y, int n) {
```

Comment:
Let's get rid of this other `median` method. Wherever it is being used, send a vector instead.